admin_ui/backend/api/system.py

```
@@ -828,73 +868,252 @@ async def _recreate_via_compose(service_name: str, health_check: bool = True):
+            pass
+        return (
+            getattr(current, "status", None) == "running",
+            getattr(current, "id", None),
+        )
+
+    def _remove_rollback_tag() -> None:
+        if not rollback_ref:
+            return
+        try:
+            client.images.remove(rollback_ref, noprune=True)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)